articles/logic-apps/error-exception-handling.md

```diff
@@ -334,6 +268,37 @@ For limits on scopes, see [Limits and config](logic-apps-limits-and-config.md).
 
 <a name="get-results-from-failures"></a>
 
+### Use "scope" in the designer
```

Comment:
Did you add these steps in the context of using scopes and "run after" for error handling? We already document the [steps to add a scope in this guide](https://learn.microsoft.com/azure/logic-apps/logic-apps-control-flow-run-steps-group-scopes), though the guide needs updates. So, we just want to describe the relevant steps for exception handling. We usually don't want to repeat already documented steps as this creates maintenance issues.

```suggestion
### Set up a scope with "run after" for exception handling
```